File: Domain/Administrator.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Domain
{
    public class Administrator
    {
        public int Id { get; set; }
        public string Email { get; set; }

        public string Name { get; set; }

        public string Password { get; set; }
    }
}

File: bettercalm-frontend/src/app/models/administrator.ts
import { Injectable } from '@angular/core';
import { Adapter } from './adapter';

export class Administrator {
    constructor(
        public id: number,
        public email: string,
        public name: string,
        public password: string
    ){}
}

@Injectable({
    providedIn: 'root'
})
export class AdministratorAdapter implements Adapter<Administrator> {
    adapt(item: any): Administrator {
        return new Administrator (
            item.id,
            item.email,
            item.name,
            item.password
        );
    }
}
